docs/application/native/guides/multimedia/sound-manager.md

```diff
@@ -192,21 +198,59 @@ To query sound device information:
 
       ```
       static void
-      _sound_device_connection_changed_cb(sound_device_h device, bool is_connected, void* user_data)
+      _sound_device_connection_changed_cb(sound_device_h device, bool is_connected, void *user_data)
       {
           int ret;
           sound_device_type_e type;
 
           if (is_connected) {
               ret = sound_manager_get_device_type(device, &type);
-              if (type == SOUND_DEVICE_BLUETOOTH)
-                  /* Connected sound device type is Bluetooth, handle accordingly */
+              if (type == SOUND_DEVICE_BLUETOOTH_MEDIA)
+                  /* Connected sound device type is Bluetooth Media, handle accordingly */
+              else
+                  /* Handle accordingly */
+          } else {
+              ret = sound_manager_get_device_type(device, &type);
+              if (type == SOUND_DEVICE_BLUETOOTH_MEDIA)
+                  /* Disconnected sound device type is Bluetooth Media, handle accordingly */
+              else
+                  /* Handle accordingly */
+          }
+      }
+      ```
+
+3. To receive a notification whenever the sound device running state changes:
+
+   1. Register a callback using the `sound_manager_add_device_running_changed_cb()` function. Use the mask to filter the callback information.
+
+      ```
+      mask = SOUND_DEVICE_IO_DIRECTION_OUT_MASK | SOUND_DEVICE_IO_DIRECTION_BOTH_MASK;
+
+      ret = sound_manager_add_device_running_changed_cb(mask, _sound_device_running_changed_cb, NULL);
+      ```
+
+      > **Note**  
```

Comment:
Please follow the Note tag style:

> **Note**
>  
> The state will be changed to **running** when the first stream goes to the device.

Reply:
Okay.

Reply:
I am sorry, if I was not clear. The note tag style is:
> ** Note**
>
> Details to be mentioned.

Reply:
Do you mean one more "empty line" between Note and the sentence?
or two spaces right after Note'xx'? (if the latter is the case, those are already there)

Reply:
Not only a space, a space with ">" tag.

> **Note**
>
> Details